### PR TITLE
Added reference on flowLogs to NetworkSecurityGroup

### PR DESCRIPTION
**Pull request by irrogozh**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json [hunk 1]
```diff
@@ -897,6 +897,14 @@
           },
           "description": "A collection of references to subnets."
         },
+        "flowLogs": {
```
Comment (lirenhe) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json]: Do you need  to update the example of this new property?

Comment (irrogozh) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json]: No, I don't think so. This property becomes populated only if there is flowLog resource created for this NSG. Plus, usually, linked properties are not specified in examples. IN NSG, besides flowLog there are a lot of linked properties (like networkInterfaces, f.e.), but none of them is in examples.

Comment (lirenhe) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json]: I see. Thanks

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json [hunk 2]
```diff
@@ -897,6 +897,14 @@
...
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "./networkWatcher.json#/definitions/FlowLog"
+          },
+          "description": "A collection of references to flow log resources."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json]: If it's a collection of references should ref type be `./network.json#/definitions/SubResource`?

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json]: Resolved offline

**Review by anton-evseev — APPROVED** [review]

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Review by lirenhe — APPROVED** [review]

**Pull request merged**